search-sentiments-map/public/scripts/map-script.js

```diff
@@ -25,14 +25,30 @@ let mapStyle = [{
 }];
 let map;
 let infowindow;
+
+// Multiplier for sentiment scores.
+const SCORE_SCALE_MULTIPLIER = 100;
+// The default score assigned to countries with not search results.
+const NO_RESULTS_DEFAULT_SCORE = -500;
+
 /* 
- * Hold the minimum and maximum values of the sentiment scores. By default, we
- * set them to hold the maximum and minimum possible integer values
- * respectively, so they can later be appropriately replaced by the actual
- * sentiment scores.
+ * Hold the minimum and maximum values of the sentiment scores.
+ * The sentiment API returns scores from -1.0 to 1.0. Our value is these max
+ * and min scores multiplied by our score multiplier.
  */
-let dataMin = Number.MAX_VALUE;
-let dataMax = Number.MIN_VALUE;
+const DATA_MIN = SCORE_SCALE_MULTIPLIER * 1.0;
+const DATA_MAX = SCORE_SCALE_MULTIPLIER  *-1.0;
+
+/**
+ * HSL color codes for country colorings.
+ * @enum {Array}
+ */
+const CountryColorCodes = {
+  GREEN: [151, 83, 34],
+  RED: [5, 69, 54],
+  DARK_GRAY: [0, 0, 31], 
+  LIGHT_GRAY: [62, 1, 83],
+};
```

Comment:
I think now would be a good time to change the overall coloring from red to blue, since we're updating the colors for countries w/o results in the branch. Similar to coloring like this, https://miro.medium.com/max/700/1*EtQD9IFqQhBXvCI5w8naxQ.png from this article https://towardsdatascience.com/the-real-world-as-seen-on-twitter-sentiment-analysis-part-two-3ed2670f927d